app/helpers/pages_helper.rb

```diff
@@ -1,2 +1,69 @@
 module PagesHelper
+   def calculate_css(xp_object)
+       ratio = xp_object.xp_min / max_xp(xp_object.level)
+
+       severity = if ratio > 0.9
+           "green-text text-accent-3"
+         elsif ratio > 0.7
+           "green-text text-darken-1"
+         elsif ratio > 0.5
+           "green-text text-lighten-3"
+         elsif ratio > 0.4
+           "green-text text-darken-3"
+         elsif ratio > 0.3
+           "indigo-text text-accent-3"
+         elsif ratio > 0.2
+           "indigo-text text-lighten-1"
+         elsif ratio > 0.1
+           "indigo-text text-lighten-2"
+         else
+           "indigo-text text-lighten-3"
+       end
+
+       severity
+     end
+
+     def max_xp(level)
```

Comment:
Admittedly it's super confusing, but the actual Max XP per level is like this: 

```2nd	300
3rd	900
4th	2,700
5th	6,500
6th	14,000
7th	23,000
8th	34,000
9th	48,000
10th	64,000
11th	85,000
12th	100,000
13th	120,000
14th	140,000
15th	165,000
16th	195,000
17th	225,000
18th	265,000
19th	305,000
20th	355,000```

You were looking at "xp to next level", this is cumulative